developerRobot/src/main/native/cpp/Robot.cpp

```diff
@@ -5,11 +5,12 @@
 #include <frc/TimedRobot.h>
 
 class Robot : public frc::TimedRobot {
+ public:
   /**
    * This function is run when the robot is first started up and should be
    * used for any initialization code.
    */
-  void RobotInit() override {}
+  Robot() : frc::TimedRobot() {}
```

Comment:
The base class default constructor is called automatically.
```suggestion
  Robot() {}
```